### PR TITLE
Improving Japanese document for better translations and Fixing typos in document

### PR DESCRIPTION
**Pull request by mackey0225**
The other day, I was thrilled to see the Japanese version of the document being provided and made public.
However, I noticed a minor issue in the translation that bothered me, so I made some corrections.

Specifically, "joining and merging" had been translated as "結合と結合," which was a repetition of the same word. I made a slight adjustment to address this.

Furthermore, while reviewing the document, I found several typos and made corrections accordingly.

### CODE REVIEW

**Comment by mackey0225** [comment]
I have read the CLA Document and I hereby sign the CLA

**Comment by mackey0225** [comment]
recheck

**Comment by jamie-lemon** [comment]
Thanks - I'm glad you appreciate the Japanese version of the docs!

I appreciate your suggestion for: 結合とマージ - makes sense to me.

If possible could you redo this PR and remove the edits to the source `.po` files. This is because the `.po` files are auto-generated from the corresponding `.rst` files and so those new amends you made to the `.rst` files which fix the typos will be auto-created when we run the generate script again. ( I also have some fixes for other rst files and intend to apply an update to address some other things ).

So, TLDR, please remove the `.po` fixes, we can merge this and then I'll auto-generate the `.po` files and add the correct `msgstr` logs against the new `msgid` items.

Hope this makes sense.

**Comment by mackey0225** [comment]
Regarding the .po file, I understand.
I made some mistakes during the corrections, such as using the wrong GitHub account for the commit, 
so I have recreated the PR #2733 .
I would appreciate it if you could review this.
